compiler/src/iree/compiler/DispatchCreation/FormDispatchRegions.cpp

```diff
@@ -712,6 +712,18 @@ fuseRootsWithProducers(MLIRContext *context, Operation *root, unsigned groupNum,
   }
 }
 
+static bool isAttentionMaskGenerator(Operation *op) {
```

Comment:
Move this into a utility function in `FusionUtils`?

Reply:
I actually didn't need this here. So it's a static function only at once place.